### DISCUSSION TITLE
[UI] Move admin-related links from the sidebar to the navbar dropdown

### DISCUSSION THREAD

**Discussion by n-klotz** [jonaswinkler/paperless-ng#637] [Feature Requests]
# Short summary 
Move the links pointing to the documentation, github and feature requests from the sidebar to the dropdown in the top right corner (see screenshot).

# Detailed description

This simple change has two advantages:
1. It frees up some "real estate" in the sidebar, which is especially useful on smaller screens with many saved views and open documents.
2. It seperates more clearly the links for accessing documents (sidebar) from links to accessing the settings (dropdown).

This only requires changing a few lines in `app-frame.component.html`, see [these commits](https://github.com/n-klotz/paperless-ng/commits/feature-navbar-links). 

However, I wanted to put it out here first for discussion before submitting a pull request. What is your opinion? 

![paperless-dropdown](https://user-images.githubusercontent.com/7721878/109187748-7d891b00-7792-11eb-894a-0f0d27e30e43.png)

**Comment by shamoon** [comment]
Yea, I dont think theres really a right or wrong answer here. I definitely considered doing this when I made the user menu (and I think we discussed it in the thread IIRC). I expect its a very familiar paradigm to most users and the first place people will look when they dont see something like "Settings" (which is why I put it in both places). The documentation links are perhaps less logical to put there and I dont think there very obtrusive in the sidebar but then again once people are comfortable with the application its hard to know how often they'll use those links (probably not so much so a +1 for putting them inside the dropdown). There is also hope of someday moving the important "Admin" stuff into the Angular front-end so the "Admin" button can be (re-)moved but given thats a significant time commitment it may not be any time soon.

One thing is that the maximum number of open documents is currently capped at 5. Im actually not sure why that is i.e. whether there would be any issues with increasing that (it would seem to make sense to increase if this change is made and if there is some issue raising it probably makes this change a little less valuable). The other thing I dont really know is the average number of saved views users have. If, like me, you have a few and rarely need > 5 documents open at once the sidebar is already ample real estate but its not hard to imagine folks with lots of saved views and / or who like to have a lot of documents open at once would benefit from making more room in the sidebar. Alas, we have no idea what the most common screen resolutions / window sizes are either.

See screenshot of my setup below just as an example, all that space seems a bit wasted (same with your screenshot too IMHO)...

TLDR: minor pros for more room in sidebar, minor cons for possibly wasted space & de-surfacing potentially useful links, wish we had a better way to gauge usefulness to paperless-ng users.

![Screen-Shot-2021-02-25-at-1 24 20-PM](https://user-images.githubusercontent.com/4887959/109220191-80254980-776d-11eb-8508-e9e9da5cae6f.jpg)
